core/sile.lua

```diff
@@ -251,11 +251,19 @@ local function suggest_luarocks (module)
 
         luarocks --lua-version %s --tree lua_modules install %s
 
-    This will install the LuaRocks to your project, then you need to tell your
-    shell to pass along that info about available LuaRocks paths to SILE. This
-    only needs to be done once in each shell.
-
-        eval $(luarocks --lua-version %s --tree lua_modules path)
+    This will install the LuaRocks to your project Note this takes advantage of
+    the fact that SILE checks for modules in the path `lua_modules` relative to
+    the current project by default. If you use this incantation, you should be
+    done.
+
+    In the event you use a different path to the LuaRocks tree, you must also
+    set environment variables to teach SILE about how to find the tree *before*
+    it runs. This can be aided by asking LuaRocks to output a path variable, but
+    it must also be suplimented to keep SILE's original knowledge of paths. This
+    only needs to be done once in each shell, (obviously substituting
```

Comment:
"once in each shell": fairly technical for lambda users. And the command below (with "sed" etc.) is fairly complex. I'm not convince we should have such long technical messages as output. It could perhaps be shortened with a link to the Wiki or another place where we could refine the wording and explanations more nicely? (Just an idea)

Reply:
I'm not worried about shortening this, but I agree the wording is overly complex. Also I forgot the `sed` incantation is *not* necessary because I already added a fix to Homebrew upstream.